modcc/cudaprinter.cpp

```diff
@@ -431,8 +417,8 @@ CUDAPrinter::CUDAPrinter(Module &m, bool o)
 
     auto proctest = [] (procedureKind k) {return k == procedureKind::api;};
```

Comment:
The name `proctest` is now rather confusing: e.g. line 420, the test reads as if we're checking if it's a procedure, then testing again if it is a procedure. Rename?